cluster/addons/fluentd-gcp/fluentd-gcp-ds.yaml

```diff
@@ -99,14 +99,7 @@ spec:
       # END_PROMETHEUS_TO_SD
       nodeSelector:
         beta.kubernetes.io/fluentd-ds-ready: "true"
-      tolerations:
-      - key: "node.alpha.kubernetes.io/ismaster"
-        effect: "NoSchedule"
```

Comment:
This actually removes the tolerations now, which it shouldn't. Please rebase & make sure they are not removed.